index.html

```diff
@@ -257,12 +281,13 @@ <h3 class="contact2">LET'S CONNECT</h3>
             <li>
               <img
                 class="scrollDown2"
-                src="portfolioimgs/down2.png"
+                src="portfolioimgs/down2.svg"
                 alt="down"
               />
             </li>
           </ul>
         </div>
+        
       </section>
 
       <section class="contact-me">
```

Comment:
- [ ] 🏃‍♂️And finally, there is some additional space that you should remove. Please kindly remove space from the left side of the contact section. And, if you complete all of these, your code will be approved ✅.

#### Your design
![image](https://user-images.githubusercontent.com/69011963/149365639-247fb4ed-a8a9-4e59-9b02-1f96f66826cc.png)